.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Please remove this `.gitignore` file from the PR. IMO, `.gitignore` should contain build related files, not user development environment files.
Check [this issue](https://github.com/golang/go/issues/23800) for a long discussion on whether `.gitignore` should/should not contain `.idea`.